monkey/monkey_island/cc/services/attack/technique_reports/T1065.py

```diff
@@ -0,0 +1,20 @@
+from monkey_island.cc.services.attack.technique_reports import AttackTechnique
+from common.utils.attack_utils import ScanStatus
+from monkey_island.cc.services.config import ConfigService
+
+__author__ = "VakarisZ"
+
+
+class T1065(AttackTechnique):
+
+    tech_id = "T1065"
+    unscanned_msg = ""
+    scanned_msg = ""
+    used_msg = ""
+    message = "Monkey used port %s to communicate to C2 server."
+
+    @staticmethod
+    def get_report_data():
+        port = ConfigService.get_config_value(['cnc', 'servers', 'current_server']).split(':')[1]
```

Comment:
IDK if code should expect server not having any port, because users can't really change the port, right?